bin.js

```diff
@@ -16,17 +20,36 @@ function print (...lines) {
 
 async function run () {
   let arg = process.argv[2] || ''
+  let filter = process.argv[3] || ''
+
   if (arg === '--version') {
     showVersion(print)
-  } else if (arg === '--help') {
+    return
+  }
```

Comment:
The code will be more readable if you will keep origin `else if` structure.

Reply:
Don't think so. In case of long if/else tree the return early pattern is better in my opinion. But it is a matter of taste. :-)

Reply:
Yeap, it is not good to change code style in PR without a rational reason